### PR TITLE
Se realizaron las validaciones de los campos del registro de usuarios.

### PR DESCRIPTION
**Pull request by EliasCode79**
Co-authored-by: Gerardo Ismael Garcia <Matarratas777>
Co-authored-by: Gaddiel Patiño <GaddPat0609>

# Description

What did you implemented/Why did you implemented this:

- The validations of the registration fields were carried out 
-  The registration button is enabled as long as you accept the terms and conditions

Examples:

- What?: Added validation of the registration fields
- Why?: With these changes the user can now register

 Example:

- Registration field validations have been added.

## Screenshots
- The registration button is not enabled if you do not fill in the fields and do not accept the terms.
image

- The fields are filled in and the terms are accepted, the button is enabled.
image

### CODE REVIEW

**Review by McFly1208 — APPROVED** [review]

**Review by IvanGodinez21 — APPROVED** [review]
Great job adding the checkbox to the validation 🐒

File: src/components/check/checkSubscribe.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Checkbox } from 'react-native-paper'
+import styles from '../cajaTextoVisible/styles'
+
+export const CheckSubscribe = ({checkout}) => {
+  // const [checked, setChecked] = React.useState(false)
+  // console.log(user)
```
Comment (YilKanda) [inline src/components/check/checkSubscribe.js:7]: Do not leave code comments on GitHub.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: src/components/check/checkSubscribe.js (hunk 1 above)
Comment (EliasCode79) [reply src/components/check/checkSubscribe.js]: We have fixed the observations you indicated, we can approve the PR.

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by HenrikhWolf — APPROVED** [review]
LGTM

**Pull request merged**